src/main/java/edu/ucsb/mapache/repositories/UserSearchRepository.java

```diff
@@ -11,5 +11,5 @@
 import java.util.Optional;
 
 public interface UserSearchRepository extends CrudRepository<UserSearch, Long> {
-
+      public List<UserSearch> findByEmail(String email);
```

Comment:
This is failing when you try to do a search, with the error:
```
2021-06-02T17:53:40.749439+00:00 app[web.1]: org.postgresql.util.PSQLException: ERROR: column "email" of relation "user_search" does not exist
```

Because you didn't also change the field in the entity to `email`

Reply:
I apologize.. I was mistaken.  There is an `email` field in the `user_search` table, but to get that to be recognized, we had to rebuild the database.